v5.17.21 (#7726)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## v5.17.21

_Jan 27, 2023_

> We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:

Reviewer: ```suggestion
We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
```

> - 🌍 Add Belarusian (be-BY) and improve some locales

Reviewer: ```suggestion
- 🌍 Add Belarusian (be-BY), Czech (cs-CZ), and Russian (ru-RU) locales
- 🌍 Improve Belarusian (be-BY), Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales
```

- 🐞 Bugfixes

### `@mui/x-data-grid@v5.17.21` / `@mui/x-data-grid-pro@v5.17.21` / `@mui/x-data-grid-premium@v5.17.21`

#### Changes

> - [DataGrid] Add `title` to cells (#7695) @thupi

Author: You had the page not refreshed may be 🙂

- [DataGrid] Fix grid state not being updated after print preview is closed (#7680) @cherniavskii
- [DataGrid] Fix non-hideable columns visibility toggling (#7716) @cherniavskii
- [DataGrid] Fix scrolling on resize for data grids inside shadow root (#7722) @cherniavskii
- [DataGridPremium] Create aggregation footer row with `isAutoGenerated: true` (#7681) @m4theushw
- [l10n] Add Belarusian (be-BY) locale (#7718) @volhalink
- [l10n] Add Slovak (sk-SK) translation for aggregation functions (#7690) @msidlo
- [l10n] Add missing core locales for `MuiTablePagination` (#7719) @MBilalShafi
- [l10n] Improve Japanese (ja-JP) locale (#7627) @makoto14
- [l10n] Improve Vietnamese (vi-VN) locale (#7601) @SpacerZ

### `@mui/x-date-pickers@v5.0.16` / `@mui/x-date-pickers-pro@v5.0.16`

#### Changes

- [pickers] Add missing components to `themeAugmentation` (#7677) @LukasTy

> - [l10n] Add Czech (cs-CZ) locale (#7666) @LukasTy

Reviewer: ```suggestion
- [l10n] Add Czech (cs-CZ) locale (#7666) @OndrejHj04
```

> - [l10n] Add Russian (ru-RU) locale (#7708) @LukasTy

Reviewer: ```suggestion
- [l10n] Add Russian (ru-RU) locale (#7708) @rstmzh
```

- [l10n] Improve Spanish (es-ES) locale (#7614) @WiXSL

> 

Reviewer: I've just merged https://github.com/mui/mui-x/pull/7723
Could we also include it in the changelog?
```suggestion

### Docs

- [docs] Add info callout about available component `slots` (#7723) @Vivek-Prajapatii
```

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## v5.17.21

_Jan 27, 2023_

We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:

- 🌍 Add Belarusian (be-BY), Czech (cs-CZ), and Russian (ru-RU) locales

> - 🌍 Improve Belarusian (be-BY), Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales

Author: ```suggestion
- 🌍 Improve Slovak (sk-SK), Japanese (ja-JP), Vietnamese (vi-VN), and Spanish (es-ES) locales
```

- 🐞 Bugfixes

### `@mui/x-data-grid@v5.17.21` / `@mui/x-data-grid-pro@v5.17.21` / `@mui/x-data-grid-premium@v5.17.21`

#### Changes

> - [DataGrid] Add `title` to cells (#7695) @thupi

Author: ```suggestion
- [DataGrid] Add `title` attribute to cells (#7695) @thupi
```
